numba/cuda/cudadrv/driver.py

```diff
@@ -37,6 +38,23 @@
 MIN_REQUIRED_CC = (2, 0)
 
 
+def _make_logger():
+    logger = logging.getLogger(__name__)
+    lvl = str(config.CUDA_LOG_LEVEL).upper()
+    lvl = getattr(logging, lvl, None)
+    if not isinstance(lvl, int):
+        lvl = logging.CRITICAL
+    logger.setLevel(lvl)
+    handler = logging.StreamHandler(sys.stderr)
+    fmt = '== CUDA [%(relativeCreated)d] %(levelname)5s -- %(message)s'
+    handler.setFormatter(logging.Formatter(fmt=fmt))
+    logger.addHandler(handler)
+    return logger
+
+
+_logger = _make_logger()
```

Comment:
There are pitfalls with configuring logging at application startup, so I think it would be better to do this lazily (for example in `Driver.__init__`).